### DISCUSSION TITLE
stammina/hunger/energy bar

### DISCUSSION THREAD

**Discussion by meoowe** [meoowe/ClickyCat#5] [Ideas] [answered]
How about adding a `Hunger bar`? This could be replenished by finding food (e.g. `cat food tins`)
We could also add a `stamina bar` that depletes every time you move. if any of these get to zero, you `lose the game`. This could also work with the #4 `Dog chasing cat`(e.g. if the dog gets to you, your stamina might go down).

**Comment by LeoNunk** [comment, marked as answer]
I don't think you should lose the game if your stamina bar gets to zero. If it does get to zero then maybe you have to wait 3 seconds for it to replenish.

**Reply by meoowe** [reply]
Great idea! That would also support #4 `Dog chasing cat`

**Comment by meoowe** [comment]
hi
